### DISCUSSION TITLE
make error. error: ‘MYSQL_OPT_SSL_ENFORCE’ was not declared in this scope

### DISCUSSION THREAD

**Discussion by shidaping** [TrinityCore/TrinityCore#28901] [Q&A]
[  0%] Built target Detour
[  0%] Built target fmt
[  4%] Built target g3dlib
[  4%] Built target sfmt
[  5%] Built target argon2
[  8%] Built target jemalloc
[  8%] Built target openssl_ed25519
[  8%] Built target gsoap
[ 10%] Built target protobuf
[ 10%] Built target revision_data.h
[ 14%] Built target common
[ 14%] Building CXX object src/server/database/CMakeFiles/database.dir/Database/MySQLConnection.cpp.o
[ 19%] Built target proto
[ 19%] Building CXX object src/server/database/CMakeFiles/database.dir/Database/QueryCallback.cpp.o
/home/wow/TrinityCore/src/server/database/Database/MySQLConnection.cpp: In member function ‘virtual uint32 MySQLConnection::Open()’:
/home/wow/TrinityCore/src/server/database/Database/MySQLConnection.cpp:148:34: error: ‘MYSQL_OPT_SSL_ENFORCE’ was not declared in this scope
  148 |         mysql_options(mysqlInit, MYSQL_OPT_SSL_ENFORCE, (char const*)&opt_use_ssl);
      |                                  ^~~~~~~~~~~~~~~~~~~~~
make[2]: *** [src/server/database/CMakeFiles/database.dir/Database/MySQLConnection.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/server/database/CMakeFiles/database.dir/all] Error 2
make: *** [all] Error 2

**Comment by Faq** [comment]
You did not provide any other information. gem version, os, ruby, rails?

**Reply by shidaping** [reply]
maridb-devel not work, it seems the version is the problem. After install mysql-client-5.7, the error gone
